cooltools/expected.py

```diff
@@ -16,6 +16,7 @@
 import bioframe
 from .lib import assign_supports, numutils
 from .lib.common import is_compatible_viewframe, is_cooler_balanced
+from .lib.common import make_cooler_view, read_viewframe
```

Comment:
hm, was it working before that ? are we using make_cooler_view and read_viewframe here  ?